subo/command/compute_deploy_core.go

```diff
@@ -179,7 +196,7 @@ func ComputeDeployCoreCommand() *cobra.Command {
 	cmd.Flags().String(branchFlag, "main", "git branch to download templates from")
 	cmd.Flags().String(versionFlag, release.SCCTag, "Docker tag to use for control plane images")
 	cmd.Flags().Int(proxyPortFlag, proxyDefaultPort, "port that the Editor proxy listens on")
-	cmd.Flags().Bool(localFlag, false, "deploy locally using docker-compose")
+	cmd.Flags().Bool(localFlag, false, "deploy locally using docker compose")
```

Comment:
Should also maybe use whatever the code decided above docker-compose to be? Though that might mean that we might need to move the code to determine whether the system has docker-compose or docker compose available to be outside / before the `if localInstall` branch.

Reply:
Or I could just use the proper noun version 😉 

```suggestion
	cmd.Flags().Bool(localFlag, false, "deploy locally using Docker Compose")
```

Reply:
This is such a niche case I'm inclined to actually do that

Reply:
...and there's precedent up on line 39:

    Long:  `deploy the Suborbital Compute Core using Kubernetes or Docker Compose`

Reply:
Which might work. Is there a way to make this super clear to our users? I'm thinking about a user who may be an experienced developer, but new to docker. Would they know the difference between `docker compose` (the command in the terminal) and `Docker Compose` (the thing they're invoking with the above command(s))?

Is there a way to make this clear / remove ambiguity?

Reply:
Inclined to leave as-is: for most users, there is no distinction (`docker compose` is present on all Docker Desktop installs updated in the last year *and* `docker-compose` is shimmed to work like an alias to `docker compose`). Pointing out the historic separation gets nuanced enough that it may actually create confusion 😆